packages/statusbar-extension/README.md

```diff
@@ -4,10 +4,8 @@ This extension creates a generic statusbar to showcase the various states of Jup
 extensions to add custom elements into the statusbar.
 
 Changing Contexts
-![Context Changes](http://g.recordit.co/OndGalRjws.gif)
 
 Component Interactions
```

Comment:
I guess these lines do not add much without the GIFs? Should we remove them?

Reply:
That's a good point, I've removed those lines too